### PR TITLE
Adds character count to reply box 😳

### LINKED ISSUE #1626: Add character count for reply box new UI
**How**
- Show characters remaining warning message only when limit reaches 50 characters or less.

Screenshot 2021-01-08 at 1 07 07 PM

@pranavrajs @sojan-official Any thoughts on the text used ?

### PR DESCRIPTION
**Pull request by gucci-ninja**
# Pull Request Template

## Description

Hey yall this is actually my first ever open source contribution. I added the remaining characters count (when it reaches 50 or below). Please review and let me know if there's any changes to be made :smiley: 

char_left

char_over

Fixes #1626 

## Type of change

UI Change

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented on my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: app/javascript/dashboard/components/widgets/WootWriter/ReplyTopPanel.vue
Comment (pranavrajs) [inline app/javascript/dashboard/components/widgets/WootWriter/ReplyTopPanel.vue:22]: @gucci-ninja This should be span. Link without href is not a good usage.

Comment (pranavrajs) [reply app/javascript/dashboard/components/widgets/WootWriter/ReplyTopPanel.vue:22]: Also please move characters remaining to i18n file.

File: app/javascript/dashboard/components/widgets/conversation/ReplyBox.vue
Comment (pranavrajs) [inline app/javascript/dashboard/components/widgets/conversation/ReplyBox.vue:121]: What happens if the message length is greater than the max allowed length. Would it show `-50 characters remaining?`

**Review by pranavrajs — CHANGES_REQUESTED** [review]
Great work on the PR! 👍  I've a couple of comments, please take a look at the individual comments.

File: app/javascript/dashboard/components/widgets/conversation/ReplyBox.vue
Comment (gucci-ninja) [reply app/javascript/dashboard/components/widgets/conversation/ReplyBox.vue:121]: yeah, i can change to `Limit exceeded`. thoughts?

**Comment by nithindavid** [comment]
@gucci-ninja Hi can you decrease the font size of the message ? Right now it's default, can you try `--font-size-mini`  as the font-size ? 
PS: If you feel it's too small and readable try `--font-size-small`.

**Comment by gucci-ninja** [comment]
@nithindavid hey do you mean the character count warning or the actual message in the reply box?

**Comment by nithindavid** [comment]
@gucci-ninja The warning message.

**Comment by gucci-ninja** [comment]
@nithindavid done!

**Comment by nithindavid** [comment]
@gucci-ninja Thanks for the effort :) I'll merge this to `develop` once the base branch is merged.
PS:  It might take couple more days.

**Comment by pranavrajs** [comment]
@gucci-ninja Sorry about this, this was a mistake. I have no clue why this was closed. The base branch was merged to develop.

**Comment by pranavrajs** [comment]
@nithindavid Can you please fix these conflicts and get it ready for merge?

**Comment by pranavrajs** [comment]
@nithindavid Please add `merge_type` to the commit message before message. 

For eg: this PR could have been `feat: Add character count to reply box`

**Pull request merged**
